examples/src/testutil_rdwr.h

```diff
@@ -206,6 +206,23 @@ int write_sync(test_cfg* cfg)
     return 0;
 }
 
+static inline
+int write_laminate(test_cfg* cfg, const char* filepath)
+{
+    /* need one process to laminate each file,
+     * we use the same process that created the file */
+    int rc = 0;
+    if (cfg->rank == 0 || cfg->io_pattern == IO_PATTERN_NN) {
+        /* laminate by setting permissions to read-only */
+        int chmod_rc = chmod(filepath, 0444);
+        if (-1 == chmod_rc) {
+            /* lamination failed */
+            rc = -1;
+        }
```

Comment:
You can replace this with just:
```C
rc = chmod(filepath, 0444);
```

Reply:
Right, I did consider that.  I decided to process the return code from chmod explicitly, since it is not intentional that the return codes for write_laminate happen to be the same as chmod().